apps/vsce/src/constants.ts

```diff
@@ -1 +1,7 @@
 export const CLI_NPM_LINK = "https://registry.npmjs.org/codemod/latest";
+export const ACCESS_TOKEN_REQUESTED_BY_VSCE_SEARCH_PARAM =
+  "accessTokenRequestedByVSCE";
+export const ACCESS_TOKEN_REQUESTED_BY_CURSOR_SEARCH_PARAM =
+  "accessTokenRequestedByCursor";
+export const VSCODE_PREFIX = "vscode://";
+export const CURSOR_PREFIX = "cursor://";
```

Comment:
With constants shared between projects, maybe it would be better to store them in one file in the root. 
and later - in order to keep things tided later lets import these into project-specific constants (so they will be taken from there in projects and not from root dir)

Reply:
Sounds good to me!